packages/google_sign_in/android/src/main/java/io/flutter/plugins/googlesignin/GoogleSignInPlugin.java

```diff
@@ -464,6 +489,15 @@ public boolean onActivityResult(int requestCode, int resultCode, Intent data) {
           } else if (pendingOperation != null && pendingOperation.method.equals(METHOD_INIT)) {
             finishWithError(ERROR_REASON_CONNECTION_FAILED, String.valueOf(resultCode));
           }
+          return true;
+        } else if (requestCode == REQUEST_CODE_RECOVER_AUTH) {
+          if (resultCode == Activity.RESULT_OK) {
+            getTokens(pendingOperation.result, (String) pendingOperation.data, false);
+            pendingOperation = null;
+          } else {
+            finishWithSuccess(null);
```

Comment:
My instinct here is that it would be better to make the result a failure. Is there a reason why you prefer a success result with null?